Elsewhen/Elements/TimeZoneChoice/TimezoneChoiceView.swift

```diff
@@ -9,10 +9,11 @@ import SwiftUI
 
 struct TimezoneChoiceView: View {
     
-    @Binding var selectedTimeZone: TimeZone
+    @Binding var selectedTimeZone: TimeZone?
     @Binding var selectedTimeZones: [TimeZone]
     @Binding var selectedDate: Date
     var selectMultiple: Bool
+    var showDeviceLocalOption: Bool = false
     var done: (() -> ())?
```

Comment:
```suggestion
    let selectMultiple: Bool
    let showDeviceLocalOption: Bool = false
    let done: (() -> ())?
```
Non-state variables probably shouldn't be mutable?